_editorial_policies/03_reviewer_information.md

```diff
@@ -0,0 +1,111 @@
+
+To ACCEPT: `link`
+To DECLINE: `link`
+
+Reviews are generally expected within 10 days, although extra time may be allowable upon request.
+
+Please be sure you review and understand our [conflict of interest policies](FILL IN LINK) before making a decision on whether or not to accept this review request.
+
+If you have any questions, please reply to this email.
+
+Thank you very much.
+`{{SENDER-FULL-NAME}}`, Associate Editor
+
+`{{MANUSCRIPT-TITLE}}`
+Abstract:
+(abstract will be automatically included)
+
+
+### Sample review action/thanks for agreeing letter
+
+(Each category has its own form letter, differing in the link to category-specific review criteria. The below sample is for Best Practices Guides)
+
+Dear XYZ,
+
+Thank you very much for agreeing to review the LiveCoMS submission, `{{MANUSCRIPT-TITLE}}`.
+Your support of our new community-run journal is very encouraging.
+
+I would appreciate receiving your review within 10 days, by `TTTTTT`.
+
+The article is to be considered under the `YYY` category.
+To aid you in your review, both the general [review criteria](https://livecomsjournal.github.io/authors/policies/#review-criteria) and the criteria specific to [Best Practices Guides](https://livecomsjournal.github.io/authors/best_practices/) are appended below.
+
+Please read our conflict-of-interest criteria and inform me of any potential conflicts.
+
+Note also that non-anonymous feedback can be provided directly to authors via the GitHub page linked in the manuscript, if you so desire.
+
+If you have any questions, please reply to this email.
+
+Thank you very much.
+`{{SENDER-FULL-NAME}}`, Associate Editor
+
+REVIEW CRITERIA
+`Fill in current general review criteria`
+`Fill in current category-specific review criteria`
+
+### Reviewer ranking questions
+
+(All fields will be communicated to the authors except comments to the editor)
+- Overall rating (one to five stars)
+- For this manuscript I recommend (Accept/Revise and resubmit/Reject)
+- How well does the manuscript meet overall review criteria and category-specific criteria?
+- Is the paper adequately edited?
+- Given that the accepted PDF will be the final version of the manuscript, are there formatting, style, or quality issues which should be addressed before acceptance? What about the quality of the figures, layout of any tables, etc.?
+- Comments to the editor
+
+
+## Sample decision letter
+
+Following is a sample decision letter which might be used for communicating the outcome of the review process to the authors.
+
+Dear `{{AUTHOR-FULL-NAME}}`,
+
+Thank you very much for your submission of your article, `{{MANUSCRIPT-TITLE}}`, to the Living Journal of Computational Molecular Sciences.
+We are delighted to inform you that we anticipate being able to accept your article, pending satisfactory completion of minor revisions as recommended by the reviewers (and detailed below).
```

Comment:
Here are some sketches for the others:
- "Your manuscript was judged to have significant merit and may ultimately be suitable for publication, but significant revisions are required.  Please see the reviewer comments below."  [continue with same letter as for minor]
- "Unfortunately, we will not be able to accept your manuscript for publication.  [editor should insert note here about whether manuscript was outside scope of journal, not of suitable quality, or otherwise inappropriate]

Reply:
Adding these.